openmetadata-service/src/main/java/org/openmetadata/service/resources/search/SearchResource.java

```diff
@@ -224,6 +224,33 @@ public Response searchBySourceUrl(
     return searchRepository.searchBySourceUrl(sourceUrl);
   }
 
+  @GET
+  @Path("/getLineage")
+  @Operation(
+      operationId = "searchLineage",
+      summary = "Search lineage",
+      responses = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "search response",
+            content = @Content(mediaType = "application/json", schema = @Schema(implementation = SearchResponse.class)))
+      })
+  public Response searchLineage(
```

Comment:
instead of keeping this inside searchResource, we should move it under /api/v1/lineage 
and introduce /api/v1/lineage/search/getLineage end point